website/pages/restaurants/[restaurant]/index.tsx

```diff
@@ -266,14 +306,16 @@ const RestaurantMenu = () => {
             </div>
           </section>
           <section className="sm:hidden">
-            <div className="scrollbar-hidden reelsContainer">
-              <div className="reel relative capitalize">
+            <div className="scrollbar-hidden mb-10 animated-fade">
+              <div
+                id="resizableRestaurantDiv"
```

Comment:
Let's use refs instead of id, as it's recommended for Vue and React for referencing DOM.